src/commands/shell.mjs

```diff
@@ -29,9 +43,18 @@ async function shellCommand(argv) {
     input: container.resolve("stdinStream"),
     eval: await buildCustomEval(argv),
     terminal: true,
+    historySize: 1000,
```

Comment:
Is this a valid argument?

I don't see it here: https://nodejs.org/docs/latest-v22.x/api/repl.html#replstartoptions

Reply:
I am now struggling to re-find it, but I thought I had a trustworthy example that included it.

I have tested and it WORKS. It seems prudent to leave some sort of limit here...

ah
> NODE_REPL_HISTORY_SIZE: Controls how many lines of history will be persisted if history is available. Must be a positive number. Default: 1000.

Since there is in fact a limit built in, I'll just remove the explicit one.